<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
<title>PaperBLAST</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
</head>
<body>
<div style="background-color: #40C0CB; display: block; position: absolute; top: 0px; left: -1px; width: 100%; padding: 0.25em; z-index: 400;"><H2 style="margin: 0em;"><A HREF="litSearch.cgi" style="color: gold; font-family: 'Montserrat', sans-serif; font-style:italic; text-shadow: 1px 1px 1px #000000; text-decoration: none;">PaperBLAST &ndash; <small>Find papers about a protein or its homologs</small></A></H2></div><P style="margin: 0em;">&nbsp;</P>
<h3>PaperBLAST Hits for Kala|Kaladp0001s0001.1 (428 a.a., MFWFYFTVAV...)</h3>
<p>Found 2 similar proteins in the literature:</p>
<p style="margin-top: 1em; margin-bottom: 0em;"><a href="http://www.uniprot.org/uniprot/Q9FZ33" title="SwissProt">Q9FZ33</a> <b>Protein AUXIN RESPONSE 4</b> from <i>Arabidopsis thaliana (Mouse-ear cress)</i> (<a href="showAlign.cgi?def1=Kala|Kaladp0001s0001.1&amp;def2=Q9FZ33&amp;seq1=MFWFYFTVAVSLVTFFFVFLSSLRNSQDPKTWFLGLPSHLRQHYAEGRTVKVQTCPNHKPIEVFTLEKGSDSSESVLIVHGLGLSSYDFRGVMAKLESKGVRVVAVDLPGSGFSEKYTIEVDEVANGIFGKLRNIYSLIREKGVFWAFDQMVDTGEIPYEEIEKLVMSRRSVKVLQLGTEEVGRVLGQVIDSLQLSPVHLVLHDSALAMSATWVVENSGSLSSVTLVDTDMNTVPAFPLWALKLPVVRELVLGLPGLYGKLMNTCCSRKINGLNMEAQRIFLKRGDGWEAVVGMGNKLNRSFDLVQWGHLKELKNVPLQVIWSGSWSEDWRNQGRQVAFGLPHARFVKHSGGRWPQSKGADEIAEGIASFISVFAKTTRQVEPEPIQKIADEAKDQIHGHHHHHDGHPHDDTPAYVEAYGLGHGWASX&amp;acc2=Q9FZ33" title="2:427/428 of query is similar to 46:472/473 of hit (E = 3e-142,  417 bits)">50% identity, 90% coverage</a>) (see <a href="http://www.ncbi.nlm.nih.gov/pubmed/16690816">paper</a>)<BR><a href="http://www.ncbi.nlm.nih.gov/protein/NP_564672.2" title="RefSeq">NP_564672</a> alpha/beta-Hydrolases superfamily protein from <i>Arabidopsis thaliana</i><BR><a href="http://www.microbesonline.org/cgi-bin/fetchLocus.cgi?locus=10083746" title="MicrobesOnline">AT1G54990</a> AXR4 (AUXIN RESISTANT 4) (RefSeq) from <i>Arabidopsis thaliana</i><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; >FUNCTION: Required for the auxin influx facilitator AUX1 polar trafficking and its asymmetric localization within the plasma membrane. Not involved in the PIN proteins localization. <BR>
DISRUPTION PHENOTYPE: Plants show reduced gravitropism, auxin- resistant root growth and AUX1 accumulation in the endoplasmic reticulum. <LI style="list-style-type: none;" margin-left: 6em; ><a href="http://doi.org/10.1126/science.1122847">AXR4 is required for localization of the auxin influx facilitator AUX1.</a><br /><small><a title="Dharmasiri S, Swarup R, Mockaitis K, Dharmasiri N, Singh SK, Kowalchyk M, Marchant A, Mills S, Sandberg G, Bennett MJ, Estelle M">Dharmasiri,</a> Science (New York, N.Y.) 2006 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/16690816">PubMed</a>)</small><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; ><a href="https://www.ncbi.nlm.nih.gov/gene/about-generif" style="color: black; text-decoration: none; font-style: italic;" title="from Gene Reference into Function (NCBI)">GeneRIF</a>: results show that AXR4 (At1g54990) a protein of the endoplasmic reticulum (ER)regulates localization of AUX1; findings indicate that the axr4 agravitropic phenotype is caused by defective AUX1 trafficking in the root epidermis</UL><LI style="list-style-type: none;" margin-left: 6em; ><a href="http://doi.org/10.1016/j.tplants.2006.09.003">Auxin and cell polarity: the emergence of AXR4.</a><br /><small><a title="Hobbie LJ">Hobbie,</a> Trends in plant science 2006 (<a href="http://www.ncbi.nlm.nih.gov/pubmed/17030147">PubMed</a>)</small><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; ><a href="https://www.ncbi.nlm.nih.gov/gene/about-generif" style="color: black; text-decoration: none; font-style: italic;" title="from Gene Reference into Function (NCBI)">GeneRIF</a>: AXR4 is crucial for the polar localization of AUX1. [AXR4]</UL><LI style="list-style-type: none;" margin-left: 6em; ><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4350080">Retraction: Mechanosensitive channel candidate MCA2 is involved in touch-induced root responses in Arabidopsis</a><br /><small><a title="Frontiers in Plant Science Editorial Office.">Frontiers,</a> Frontiers in plant science 2015 </small><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; >&ldquo;...in the Col-0 wild type, and found that the expression level of the AXR4 gene (<B><span style="color: red;">At1G54990</span></B>), which encodes a protein required for the subcellular localization of the auxin influx carrier AUX1 (Dharmasiri et al., 2006 ), was significantly lower in the mca2 -null mutant. To confirm...&rdquo;</UL><LI style="list-style-type: none;" margin-left: 6em; ><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4205128">Microsome-associated proteome modifications of Arabidopsis seedlings grown on board the International Space Station reveal the possible effect on plants of space stresses other than microgravity</a><br /><small><a title="Mazars C, BriÃ¨re C, Grat S, Pichereaux C, Rossignol M, Pereda-Loth V, Eche B, Boucheron-Dubuisson E, Le Disquet I, Medina FJ, Graziana A, Carnero-Diaz E.">Mazars,</a> Plant signaling & behavior 2014 </small><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; >&ldquo;...hydratase/3-hydroxyacyl-CoA dehydrogenase Q8LAZ1 1.681 very-long-chain fatty acid condensing enzyme CUT1 Auxin metabolism and trafficking Q9FZ33 <B><span style="color: red;">AT1G54990</span></B> 2.909 protein auxin RESPONSE 4 Q9SRU2 AT3G02260 2.471 Auxin transport protein BIG Q9S9W2 AT4G05530 2.449 dehydrogenase/reductase SDR family member 4 Q0WLB5 AT3G08530 1.744 Clathrin, heavy chain Cell Wall Q1JPL7 AT1G11580...&rdquo;</UL><LI style="list-style-type: none;" margin-left: 6em; ><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4455775">Abscisic acid regulates root elongation through the activities of auxin and ethylene in Arabidopsis thaliana</a><br /><small><a title="Thole JM, Beisner ER, Liu J, Venkova SV, Strader LC.">Thole,</a> G3 (Bethesda, Md.) 2014 </small><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; >&ldquo;...linkage between ABA-resistant root elongation and the identified mutations. Genotyping for the AUXIN RESISTANT4 / <B><span style="color: red;">At1g54990</span></B> ( AXR4 ) ( Dharmasiri et al. 2006 ) mutation revealed that all of the individuals tested were homozygous at the AXR4 locus, suggesting this may be the locus responsible...&rdquo;<LI style="list-style-type: none;" margin-left: 6em; >&ldquo;...for a subset of identified mutations and used to localize the AR241 causative mutation near <B><span style="color: red;">At1g54990</span></B> with 0/96 recombination events in a population of AR241 backcrossed lines. Recombination events in examined backcrossed lines are listed to the right of gene names for each PCR-based marker. (B)...&rdquo;</UL></UL></p>
<p style="margin-top: 1em; margin-bottom: 0em;"><a href="http://www.ncbi.nlm.nih.gov/protein/XP_002375078.1" title="RefSeq">AFLA_035680</a> hydrolase, putative from <i>Aspergillus flavus NRRL3357</i> (<a href="showAlign.cgi?def1=Kala|Kaladp0001s0001.1&amp;def2=AFLA_035680&amp;seq1=MFWFYFTVAVSLVTFFFVFLSSLRNSQDPKTWFLGLPSHLRQHYAEGRTVKVQTCPNHKPIEVFTLEKGSDSSESVLIVHGLGLSSYDFRGVMAKLESKGVRVVAVDLPGSGFSEKYTIEVDEVANGIFGKLRNIYSLIREKGVFWAFDQMVDTGEIPYEEIEKLVMSRRSVKVLQLGTEEVGRVLGQVIDSLQLSPVHLVLHDSALAMSATWVVENSGSLSSVTLVDTDMNTVPAFPLWALKLPVVRELVLGLPGLYGKLMNTCCSRKINGLNMEAQRIFLKRGDGWEAVVGMGNKLNRSFDLVQWGHLKELKNVPLQVIWSGSWSEDWRNQGRQVAFGLPHARFVKHSGGRWPQSKGADEIAEGIASFISVFAKTTRQVEPEPIQKIADEAKDQIHGHHHHHDGHPHDDTPAYVEAYGLGHGWASX&amp;acc2=XP_002375078.1" title="63:135/428 of query is similar to 18:89/292 of hit (E = 8e-05, 47.8 bits)">37% identity, 17% coverage</a>)<UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; ><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3877130">MIDDAS-M: motif-independent de novo detection of secondary metabolite gene clusters through the integration of genome sequencing and transcriptome data</a><br /><small><a title="Umemura M, Koike H, Nagano N, Ishii T, Kawano J, Yamane N, Kozone I, Horimoto K, Shin-ya K, Asai K, Yu J, Bennett JW, Machida M.">Umemura,</a> PloS one 2013 </small><UL style="margin-top: 0em; margin-bottom: 0em;"><LI style="list-style-type: none;" margin-left: 6em; >&ldquo;...example, the peak circled in Fig. 4B detected only in a limited conditions, composed of <B><span style="color: red;">AFLA_035680</span></B> through AFLA_035720, was not detected either by SMURF or by antiSMASH. 10.1371/journal.pone.0084028.g004 Figure 4 SMB gene cluster detection by MIDDAS-M in A. flavus . (A) A 3D view of the...&rdquo;</UL></UL></p>
<script src="http://fit.genomics.lbl.gov/d3js/d3.min.js"></script>
             <script src="http://fit.genomics.lbl.gov/images/fitblast.js"></script>
             <H3><A title="Fitness BLAST searches for similarity to bacterial proteins that have mutant phenotypes" HREF="http://fit.genomics.lbl.gov/" NAME="#fitness">Fitness Blast</A></H3>
             <P><DIV ID="fitblast_short"></DIV></P>
             <script>
             var server_root = "http://fit.genomics.lbl.gov/";
             var seq = "MFWFYFTVAVSLVTFFFVFLSSLRNSQDPKTWFLGLPSHLRQHYAEGRTVKVQTCPNHKPIEVFTLEKGSDSSESVLIVHGLGLSSYDFRGVMAKLESKGVRVVAVDLPGSGFSEKYTIEVDEVANGIFGKLRNIYSLIREKGVFWAFDQMVDTGEIPYEEIEKLVMSRRSVKVLQLGTEEVGRVLGQVIDSLQLSPVHLVLHDSALAMSATWVVENSGSLSSVTLVDTDMNTVPAFPLWALKLPVVRELVLGLPGLYGKLMNTCCSRKINGLNMEAQRIFLKRGDGWEAVVGMGNKLNRSFDLVQWGHLKELKNVPLQVIWSGSWSEDWRNQGRQVAFGLPHARFVKHSGGRWPQSKGADEIAEGIASFISVFAKTTRQVEPEPIQKIADEAKDQIHGHHHHHDGHPHDDTPAYVEAYGLGHGWASX";
             fitblast_load_short("fitblast_short", server_root, seq);
             </script>
    <h3>Query Sequence</h3><p style="font-family: monospace;"><small>>Kala|Kaladp0001s0001.1<br />MFWFYFTVAVSLVTFFFVFLSSLRNSQDPKTWFLGLPSHLRQHYAEGRTVKVQTCPNHKP<br />IEVFTLEKGSDSSESVLIVHGLGLSSYDFRGVMAKLESKGVRVVAVDLPGSGFSEKYTIE<br />VDEVANGIFGKLRNIYSLIREKGVFWAFDQMVDTGEIPYEEIEKLVMSRRSVKVLQLGTE<br />EVGRVLGQVIDSLQLSPVHLVLHDSALAMSATWVVENSGSLSSVTLVDTDMNTVPAFPLW<br />ALKLPVVRELVLGLPGLYGKLMNTCCSRKINGLNMEAQRIFLKRGDGWEAVVGMGNKLNR<br />SFDLVQWGHLKELKNVPLQVIWSGSWSEDWRNQGRQVAFGLPHARFVKHSGGRWPQSKGA<br />DEIAEGIASFISVFAKTTRQVEPEPIQKIADEAKDQIHGHHHHHDGHPHDDTPAYVEAYG<br />LGHGWASX</small></p><h3><a href="litSearch.cgi">New Search</a></h3>
<H3><A NAME="stats">Statistics</A></H3>

The PaperBLAST database links 401,049 different protein sequences to 723,768 scientific articles. Searches against EuropePMC were last performed on March 26 2017.

<H3><A NAME="works">How It Works</A></H3>

<P>PaperBLAST builds a database of protein sequences that are linked
to scientific articles. These links come from automated text searches
against the articles in <A HREF="http://europepmc.org/">EuropePMC</A>
and from manually-curated information from <A
HREF="https://www.ncbi.nlm.nih.gov/gene/about-generif" title="Gene
Reference into Function (NCBI)">GeneRIF</A>, <A
HREF="http://www.uniprot.org/">Swiss-Prot</A>, and <A
HREF="http://ecocyc.org">EcoCyc</A>.  As of March 2017, PaperBLAST
links over 360,000 different protein sequences to over 700,000
articles.  Given this database and a protein sequence query,
PaperBLAST uses <A
HREF="https://en.wikipedia.org/wiki/BLAST">protein-protein BLAST</A>
to find similar sequences with E &lt; 0.001.

<P>To build the database, we query EuropePMC with locus tags, with <A
HREF="https://www.ncbi.nlm.nih.gov/refseq/">RefSeq</A> protein
identifiers, and with <A HREF="http://www.uniprot.org/">UniProt</A>
accessions.  We obtain the locus tags from RefSeq or from <A
HREF="http://www.microbesonline.org/">MicrobesOnline</A>.  We use
queries of the form "locus_tag AND genus_name" to try to ensure that
the paper is actually discussing that gene.  Because EuropePMC indexes
most recent biomedical papers, even if they are not open access, some
of the links may be to papers that you cannot read or that our
computers cannot read.  We query each of these identifiers that
appears in the open access part of EuropePMC, as well as every locus
tag that appears in the 500 most-referenced genomes, so that a gene
may appear in the PaperBLAST results even though none of the papers
that mention it are open access. We also incorporate text mined links
from EuropePMC that link open access articles to UniProt or RefSeq
identifiers.  (This yields some additional links because EuropePMC
uses different heuristics for their text mining than we do.)

<P>For every article that mentions a locus tag, a RefSeq protein
identifier, or a UniProt accession, we try to select one or two
snippets of text that refer to the protein. If we cannot get access to
the full text, we try to select a snippet from the abstract, but
unfortunately, unique identifiers such as locus tags are rarely
provided in abstracts.

<P>We also use manually-curated links between protein sequences and
articles: <UL> <LI>We index proteins from NCBI's RefSeq if a GeneRIF
entries links the gene to an article in <A
HREF="http://www.ncbi.nlm.nih.gov/pubmed/">PubMed</A><sup>&reg;</sup>.
GeneRIF also provides a short summary of the article's claim about the
protein, which we provide instead of a snippet.  <LI>We index proteins
from Swiss-Prot (the curated part of UniProt) if the curators
identified experimental evidence for the protein's function (evidence
code ECO:0000269).  <LI>We index every protein EcoCyc, a curated
database of the proteins in <i> Escherichia coli</i> K-12.  </UL> For
the entries from Swiss-Prot and EcoCyc, we provide a short curated
description of the protein's function.  Most of these entries also
link to articles in <A
HREF="http://www.ncbi.nlm.nih.gov/pubmed/">PubMed</A>.

<P>For more information see the <A HREF="http://biorxiv.org/content/early/2017/05/02/133041">preprint</A> or the
<A
HREF="https://github.com/morgannprice/PaperBLAST">code</A>.

<H3><A NAME="secret">Secrets</A></H3>

<P>PaperBLAST cannot provide snippets for many of the papers that are
published in non-open-access journals. This limitation applies even if
the paper is marked as "free" on the publisher's web site and is
available in PubmedCentral or EuropePMC. If a journal that you publish
in is marked as "secret," please consider publishing elsewhere.

<H3><A NAME="omission">Omissions from the PaperBLAST Database</A></H3>

<P>Some important articles are missing from PaperBLAST, either because
the article's full text is not in EuropePMC (as for many older
articles) or because of PaperBLAST's heuristics. If you notice an
article that characterizes a protein's function but is missing from
PaperBLAST, please notify the curators at <A
HREF="http://www.uniprot.org/update">UniProt</A> or add an entry to <A
HREF="https://www.ncbi.nlm.nih.gov/gene/submit-generif">GeneRIF</A>.
Entries in either of these databases will eventually be incorporated
into PaperBLAST.  Note that to add an entry to UniProt, you will need
find the UniProt identifier for the protein.  If the protein is not
already in UniProt, you can ask them to create an entry.  To add an
entry to GeneRIF, you will need an NCBI Gene identifier, but
unfortunately many prokaryotic proteins in RefSeq do not have
corresponding Gene identifers.

<center>by <A HREF="http://morgannprice.org/">Morgan Price</A>,
<A HREF="http://genomics.lbl.gov/">Arkin group</A><BR>
Lawrence Berkeley National Laboratory
</center>

</body>
</html>